vite.config.js

```diff
@@ -66,7 +66,7 @@ function wordPressExternals() {
 					match;
 				const imports = defaultImport || namedImports;
 
-				if ( module.match( /\.css\?inline$/ ) ) {
+				if ( module.match( /\.css\?(inline|raw)$/ ) ) {
```

Comment:
Added for robustness and avoid cryptic. There is no active use of Vite's `raw` query parameter at the present.